pipeline/intracellular.py

```diff
@@ -0,0 +1,158 @@
+        sess_data_dir = os.path.join('..', 'data', 'whole_cell_nwb2.0')
+        # Get the Session definition from the keys of this session
+        animal_id = key['subject_id']
+        date_of_experiment = key['session_time']
+        # Search the files in filenames to find a match for "this" session (based on key)
+        sess_data_file = utilities.find_session_matched_nwbfile(sess_data_dir, animal_id, date_of_experiment)
+        if sess_data_file is None:
+            print(f'IntracellularAcquisition import failed for: {animal_id} - {date_of_experiment}')
+            return
+        nwb = h5.File(os.path.join(sess_data_dir, sess_data_file), 'r')
+        #  ============= Now read the data and start ingesting =============
+        print('Insert intracellular data for: subject: {0} - date: {1} - cell: {2}'.format(key['subject_id'],
+                                                                                           key['session_time'],
+                                                                                           key['cell_id']))
+        # -- MembranePotential
+        membrane_potential_time_stamps = nwb['acquisition']['timeseries']['membrane_potential']['timestamps'].value
+        self.insert1(dict(key,
+                                            membrane_potential = nwb['acquisition']['timeseries']['membrane_potential'][
+                                                'data'].value,
+                                            membrane_potential_wo_spike =
+                                            nwb['analysis']['Vm_wo_spikes']['membrane_potential_wo_spike'][
+                                                'data'].value,
+                                            membrane_potential_start_time = membrane_potential_time_stamps[0],
+                                            membrane_potential_sampling_rate = 1 / np.mean(
+                                                np.diff(membrane_potential_time_stamps))))
+        nwb.close()
+
+
+@schema
+class CurrentInjection(dj.Imported):
+    definition = """
+    -> Cell
+    ---
+    current_injection: longblob
+    current_injection_start_time: float  # first timepoint of current injection recording
+    current_injection_sampling_rate: float  # (Hz) sampling rate of current injection recording
+    """
+
+    def make(self, key):
+        # ============ Dataset ============
+        sess_data_dir = os.path.join('..', 'data', 'whole_cell_nwb2.0')
+        # Get the Session definition from the keys of this session
+        animal_id = key['subject_id']
+        date_of_experiment = key['session_time']
+        # Search the files in filenames to find a match for "this" session (based on key)
+        sess_data_file = utilities.find_session_matched_nwbfile(sess_data_dir, animal_id, date_of_experiment)
+        if sess_data_file is None:
+            print(f'IntracellularAcquisition import failed for: {animal_id} - {date_of_experiment}')
```

Comment:
Do not suppress errors from `make`.  Use `populate(..., suppress_errors=True)` instead.